README.md

```diff
@@ -0,0 +1,25 @@
+*******
+UnitBot
+*******
+
+
+`unitBot` is a slack bot that convert imperial unit into european unit, and vice
+versa.
+
+To run one need pivate.py file which contains (see example below):
+- api token,
+- a user name or channel name to send the error messages.
+
+``` API_TOKEN= 'my_slack_group_token' ERRORS_TO= 'where_to_send_erorr' ```
+
```

Comment:
Maybe include here that users could also provide the API token in a `SLACKBOT_API_TOKEN` environment variable.